source/docs/articles/drupal/8/configuration-management.md

```diff
@@ -32,13 +32,13 @@ This keeps us from needing to use `--site=book-blog` with all of our terminus co
 
 ### Workflow Example
 
-1. `terminus drush cex -y --env=dev`
+1. `terminus drush 'cex -y --env=dev'`
```

Comment:
This is not correct. the `--env=dev` piece of this belongs to Terminus, not Drush, so it should not be inside the quotes. It also needs a `--site=<site>` It should be
`terminus drush 'cex -y' --env=dev --site=<site>`